stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional, Union
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[str]]: ...
+    def list_directory(self, path: AnyStr) -> Optional[StringIO]: ...
+    def translate_path(self, path: AnyStr) -> str: ...
```

Comment:
The return value should also be AnyStr. In general, a single AnyStr doesn't make sense, because the point of the AnyStr type variable is to constrain two types to be the same (either str or unicode).

As far as I'm aware, using only AnyStr in a signature is meaningful only when it's the type argument to a generic class like List. In that case, `List[AnyStr]` means the list must either have only str or only unicode.